Src/Campus.Master.API/UI/src/app/modules/auth/registration/registration.component.ts

```diff
@@ -12,7 +12,7 @@ import { HttpErrorResponse } from '@angular/common/http';
 import { Subscription, zip } from 'rxjs';
 import { RegisterUser } from '../shared/models/register-user';
 import { StateTransfer } from '@shared/models/state-transfer';
-import { LocaleService } from '../../shared/services';
+import { LocaleService } from '../../../shared/services';
```

Comment:
Faced the same issue in #65. Any ideas how to fix this?

Reply:
Keep researching